app/controllers/Subscriptions.scala

```diff
@@ -124,6 +126,12 @@ class Subscriptions(
     Ok(views.html.main(title, id, js, css, description)).withSettingsSurrogateKey
   }
 
+  def paperPrices(): Action[AnyContent] = NoCacheAction() { implicit request =>
```

Comment:
I would think it's worth making this response cacheable

Reply:
Good point! two things on that
1. There's a standing question of how stale can we allow the prices to get (@paulbrown1982 ?)
2. Should we be okay to make this cacheable we can include the json within the paper subs page vs making an endpoint altogether

Reply:
I think including it in the page would be preferable, otherwise we have to cater for the possibility of the pricing call failing. I can't see staleness being an issue as we only cache for 1 minute